tasks/main.yml

```diff
@@ -11,6 +11,17 @@
     state: "{{ docker_package_state }}"
   notify: restart docker
 
+- name: Ensure /etc/docker exists
+  file:
+    path: /etc/docker
+    state: directory
+
+- name: Install daemon.json
```

Comment:
This task should have a variable to control whether it actually is done or not; some people may already be controlling their `daemon.json` file differently. And since this role doesn't _currently_ manage the file, the default should be to not manage the file.

So, something like `docker_daemon_json_manage: false` in `defaults/main.yml` and documented in the README.

Reply:
What do you think if we don't introduce a new variable and instead, invoke the tasks if `docker_config_options` is defined and length>0 instead ?